sphinx/writers/latex.py

```diff
@@ -399,6 +401,15 @@ def __init__(self, document, builder):
         if self.elements['polyglossia']:
             self.elements['babel'] = ''  # disable babel forcely
             self.elements['multilingual'] = self.elements['polyglossia']
+            self.elements['classoptions'] += ',' + self.babel.get_language()
```

Comment:
I don't know (will check): but other packages may use this option. A documentclass option is presented to all packages and packages like "datetime/datetime2" use it.